test/test-framework/build/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ENV OPERATOR=/usr/local/bin/memcached-operator \
+    USER_UID=1001 \
+    USER_NAME=memcached-operator
+
+# install operator binary
+COPY build/_output/bin/memcached-operator ${OPERATOR}
+
+COPY build/bin /usr/local/bin
+RUN  /usr/local/bin/user_setup
+
+ENTRYPOINT ["/usr/local/bin/entrypoint"]
+
+USER ${USER_UID}
```

Comment:
This file looks new and has no other references in this PR. Does it have a purpose? It seems unrelated to the PR's goal of checking for API rule violations.

Reply:
The test-framework is the Memcached Getting Started project + one additional kind. We need to be able to generate the data inside of it by using the operator-sdk commands.  So, since to run `operator-sdk generate k8s and openapi` it needs to check if is in the root of the project and this check is done by validating if the build/Dockerfile exists then it should be here. Without it we cannot update the mock data using the tool. 

Is it make sense? Please, let me know if I could properly explain why I added it here and in this PR.